doc/source/configuration/magnum-capi.rst

```diff
@@ -0,0 +1,36 @@
+=========================
+Magnum Cluster API Driver
+=========================
+
```

Comment:
Intro:
what is the new driver?
how does it relate to the old driver?
can I use both together?

Reply:
are changes sufficient?

Reply:
LGTM, thanks